### ISSUE TITLE
Content of CITATION file

### ISSUE THREAD

**Issue by c-martinez** [NLeSC/guide#134]
Following on the discussion from #132 

The guide provides guidelines for what information should be provided with your software release, but is not necessarily prescribing the format for that information. So from my point of view, a CITATION file could contain the following information:

```
If you are using this software, please cite our paper:
DOI: XXX

Citation is provided in Bibtex & cff format.
```

Then your repo could also contain the Bibtex, cff or whatever other format is currently best for providing citations.

**Comment by bouweandela** [comment]
We already have a description of the CITATION file here:
https://github.com/NLeSC/guide/blob/master/software/documentation.md#citation-file

**Comment by c-martinez** [comment]
Did link move to here https://github.com/NLeSC/guide/blob/master/best_practices/documentation.md#citation-file ?

Cool! I'm not sure the link provides sufficient information on how to properly build your CITATION file. But maybe that should be the topic of a different issue?

**Comment by bouweandela** [comment]
Indeed. Maybe you could just add any information you feel is missing to that section and then we can close this issue? Maybe this could be a useful suggestion? https://citation-file-format.github.io

**Comment by c-martinez** [comment]
Can be closed once #147 is merged.
